app/scripts/modules/cloudfoundry/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "jsx": "react",
-    "lib": ["es2017", "dom"],
+    "lib": ["es2019", "dom"],
```

Comment:
does typescript emit a polyfill for this, or does it emit es2019 features directly (requiring browser support)?

Reply:
@christopherthielen I believe that this will add support for type checking es2019, leave the compilation target at es6, and will not polyfill  for the 2019 spec. After looking at the webpack config, it seems we are also not polyfilling the existing 2017 spec or using babel for tsx files. Is there somewhere else where we are applying polyfills?

Reply:
Chatted with @christopherthielen and confirmed it is not getting polyfilled. I'm going to replace the es2019 usage with lodash.